qiskit/pulse/builder.py

```diff
@@ -1055,7 +1056,7 @@ def align_func(
 
     Examples:
 
-    .. jupyter-execute::
+    .. code-block:: python
```

Comment:
Why this change? Because of deprecation?

Reply:
This is no longer relevant but you cannot build document with jupyter-execute that executes deprecated code (because warning is handled as error).